Tests/Unit/Stubs/Microsoft365.psm1

```diff
@@ -17179,7 +17179,8 @@ function Update-MgDeviceManagementIntent
 function Update-MgDeviceManagementIntentSetting
```

Comment:
Just as an FYI, this file is automatically generated on a regular basis by the New-M365DSCStub cmdlet. There is a chance that any manual change get overwritten.